docs/usage/configuration-options.md

```diff
@@ -2517,7 +2514,15 @@ The managers listed above all have this default registryAlias:
 }
 ```
 
-Alias values must be properly formatted URIs.
+For the `gitlabci` manager it can be useful to configure an alias for the `$CI_REGISTRY` variable:
+
+```json
+{
+  "registryAliases": {
+    "$CI_REGISTRY": "registry.example.com"
+  }
+}
+```
```

Comment:
Should we move this explanation to the [`gitlabci` manager docs](https://docs.renovatebot.com/modules/manager/gitlabci/)?

Source: [`lib/modules/manager/gitlabci/readme.md`](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/gitlabci/readme.md)

Reply:
Not sure, we also have the Helm content here 🤔

Reply:
See https://github.com/renovatebot/renovate/pull/16227#discussion_r912949581